test/boost/row_cache_test.cc

```diff
@@ -3360,8 +3360,6 @@ SEASTAR_TEST_CASE(test_concurrent_reads_and_eviction) {
         auto pr = dht::partition_range::make_singular(m0.decorated_key());
         auto make_reader = [&] (const query::partition_slice& slice) {
             auto rd = cache.make_reader(s, semaphore.make_permit(), pr, slice);
-            rd.set_max_buffer_size(3);
```

Comment:
Should we do the same fix in [test_readers_get_all_data_after_eviction](https://github.com/scylladb/scylla/blob/29fcfe5b1c93406c87c2deeeac8b05933be2cb24/test/boost/row_cache_test.cc#L2278-L2279), [test_tombstones_are_not_missed_when_range_is_invalidated](https://github.com/scylladb/scylla/blob/29fcfe5b1c93406c87c2deeeac8b05933be2cb24/test/boost/row_cache_test.cc#L2407-L2408), [test_exception_safety_of_update_from_memtable](https://github.com/scylladb/scylla/blob/29fcfe5b1c93406c87c2deeeac8b05933be2cb24/test/boost/row_cache_test.cc#L2512-L2513), [test_concurrent_population_before_latest_version_iterator](https://github.com/scylladb/scylla/blob/29fcfe5b1c93406c87c2deeeac8b05933be2cb24/test/boost/row_cache_test.cc#L2725-L2726), [test_random_row_population](https://github.com/scylladb/scylla/blob/29fcfe5b1c93406c87c2deeeac8b05933be2cb24/test/boost/row_cache_test.cc#L2889-L2890), [test_continuity_is_populated_when_read_overlaps_with_older_version?](https://github.com/scylladb/scylla/blob/29fcfe5b1c93406c87c2deeeac8b05933be2cb24/test/boost/row_cache_test.cc#L3015-L3016), [test_continuity_population_with_multicolumn_clustering_key](https://github.com/scylladb/scylla/blob/29fcfe5b1c93406c87c2deeeac8b05933be2cb24/test/boost/row_cache_test.cc#L3147-L3148), [test_tombstone_merging_of_overlapping_tombstones_in_many_versions](https://github.com/scylladb/scylla/blob/29fcfe5b1c93406c87c2deeeac8b05933be2cb24/test/boost/row_cache_test.cc#L3323-L3324) and [test_random_population_with_many_versions](https://github.com/scylladb/scylla/blob/29fcfe5b1c93406c87c2deeeac8b05933be2cb24/test/boost/row_cache_test.cc#L3658-L3659) ?

Reply:
Yes